packages/@o3r/telemetry/src/environment/index.ts

```diff
@@ -1,6 +1,7 @@
 import * as fs from 'node:fs';
 import * as os from 'node:os';
 import { execSync } from 'node:child_process';
+import path from 'node:path';
```

Comment:
```suggestion
import * as path from 'node:path';
```
to avoid simplified import as much as possible